serverless/networking/serverless-ossm-jwt.adoc

```diff
@@ -0,0 +1,101 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-jwt"]
+= Using JSON Web Token authentication with {ProductShortName} and {ServerlessProductName}
+:context: serverless-ossm-jwt
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This section provides information about how you can enable JSON Web Token (JWT) authentication for Knative services.
+
+.Prerequisites
+* Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+* Install xref:../../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[{ProductName}].
+* Configure xref:../../serverless/networking/serverless-ossm.adoc#serverless-ossm[{ProductShortName} with {ServerlessProductName}], including enabling sidecar injection for your Knative services.
+
+[IMPORTANT]
+====
+Adding sidecar injection to Pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
+====
+
+.Procedure
+
+. Create a policy in your serverless application namespace that only allows requests with valid JSON Web Tokens (JWT):
+
+.. Copy the following Policy resource into a YAML file:
++
+
+[source,yaml]
+----
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: default
+spec:
+  origins:
+  - jwt:
+      issuer: testing@secure.istio.io
+      jwksUri: "https://raw.githubusercontent.com/istio/istio/release-1.6/security/tools/jwt/samples/jwks.json"
+      triggerRules:
+      - excludedPaths:
+        - prefix: /metrics
+        - prefix: /healthz
+  principalBinding: USE_ORIGIN
+----
+
++
+[IMPORTANT]
+====
+The paths `/metrics` and `/healthz` must be included in `excludedPaths` because they are accessed from system Pods in the `knative-serving` namespace.
+====
+
+.. Apply the Policy resource YAML file:
```

Comment:
This step is rendering as `a.` instead of `b.` in the preview.

Reply:
The preview likely is from the original commit and therefore outdated. Ashley said that moving the note fixed it. Would be a good idea to double-check with a local asciibinder build, @abrennan89